libpromises/generic_agent.c

```diff
@@ -1040,6 +1066,13 @@ void GenericAgentInitialize(EvalContext *ctx, GenericAgentConfig *config)
         FatalError(ctx, "Error determining working directory");
     }
 
+#ifndef __MINGW32__
+    if (!SetGroupToSystemGroup())
```

Comment:
I think the *does nothing on Windows* logic should be inside the function.

Reply:
and return true in windows case?

Reply:
Yes